nlp/environment.yml

```diff
@@ -1,9 +1,18 @@
 name: birdy
+
 channels:
-  - conda-forge
-  - defaults
+- defaults
+- conda-forge
+
 dependencies:
-  - intake-esm
-  - intake-stac
-  - sat-search
-  - threddsclient
+- python-flair
+- spacy==3.0.6
+- numpy<1.20.0
+- python-dateutil
+- python-levenshtein==0.12.2
+- requests=2.25.1
+- pip==20.3.3
+
+- pip:
+  - textsearch==0.0.21
+  - spacy 
```

Comment:
why is this package also installed through `RUN python -m spacy download en_core_web_trf`?

Reply:
This is a weird problem, on the spacy documentation site : https://spacy.io/usage
they only mention the installation of a model through `python -m spacy download <model>`
I was able to put it in the environment.yml like: 
`
dependencies : 
  - pip:
      - spacy==3.1.0
  - spacy-model-en_core_web_sm 
`
But I dont know if it is a version issue, but `en_core_web_sm` model works in the environment.yml but not the one needed --> `en_core_web_trf`. 
Even when forcing the latest verison `spacy==3.1.0` (https://spacy.io/models/en#en_core_web_trf)
It only works throught the  `python -m spacy download <model>` for the `trf` model.

Reply:
Weird.. Ok, then if it is an actual limitation, please add a comment on top of the `python -m spacy download <model>` instruction, to explain why it's like that. In this case we won't ask ourselves this question when we'll see this duplicate.